projects/igniteui-angular/schematics/ng-add/index.ts

```diff
@@ -49,40 +39,46 @@ const enableWebAnimationsAndGridSupport = (tree: Tree, targetFile: string, polyf
   tree.overwrite(targetFile, polyfillsData);
 };
 
-const readInput = (options: Options): Rule => (tree: Tree, context: SchematicContext) => {
+const readInput = (options: Options): Rule =>
+  async (tree: Tree, context: SchematicContext) => {
+    const workspaceHost = createHost(tree);
+    const { workspace } = await workspaces.readWorkspace(tree.root.path, workspaceHost);
     if (options.polyfills) {
-        const workspace = getWorkspace(tree);
-        const targetProperty = 'es5BrowserSupport';
-        const project = workspace.projects[workspace.defaultProject];
-        const polyfillsFile = getConfigFile(project, 'polyfills');
-        const propertyExists = propertyExistsInWorkspace(targetProperty, workspace);
-        let polyfillsData = tree.read(polyfillsFile).toString();
-        if (propertyExists) {
+      const targetProperty = 'es5BrowserSupport';
```

Comment:
I believe this is no longer in use and this functionality probably doesn't do what it's supposed to